ci/gpu/build.sh

```diff
@@ -58,7 +58,7 @@ gpuci_mamba_retry install -y \
                "cugraph=$MINOR_VERSION.*" \
                "cuspatial=$MINOR_VERSION.*" \
                "dask-cudf=$MINOR_VERSION.*" "dask-cuda=$MINOR_VERSION.*" \
-               "numba>=0.51.2" \
+               "numba>=0.54" \
```

Comment:
@AjayThorve, can you also bump the `numba` version in the `integration` PR?

Reply:
@ajschmidt8 actually its `0.53.1` in some places and `0.54` in some throughout the [rapidsai org](https://github.com/search?q=org%3Arapidsai+numba%3E%3D0.54&type=code). 

Should I still update it to `0.54` or should I update cuxfilter dependency to `>=0.53.1`?

Reply:
@AjayThorve, you can update to `0.54` in the `integration` PR. `0.54` is required by `ptxcompiler` ([link](https://github.com/rapidsai/ptxcompiler/blob/b2148e6f98b9de312bb3cf952da13c026e98023c/conda/recipes/ptxcompiler/meta.yaml#L25)) for CUDA Enhanced Compatibility, so that's the lowest version we should be using across the board.

Reply:
done

Reply:
Great. Once CI passes there, we can merge that PR followed by this one :+1:

Reply:
Sounds good, thanks!